pyproject.toml

```diff
@@ -126,6 +127,7 @@ documentation = "https://docs.saleor.io/"
   types-six = "^1.16.17"
   vcrpy = ">=4.0,<5.0"
   fakeredis = "^2.10"
+  trustme = "^1.1.0"
```

Comment:
trustme is a leftover which was used by tests but they got moved over here: https://github.com/saleor/requests-hardened/blob/20b8b9a067e1da646736b6d349ca1d11dac7ead7/tests/http_test_servers/tls.py#L63

I will remove it

Reply:
Resolved at 0add19d7438e9971ce1f5f4ab7b62f7cd23dfedf